src/costab.c

```diff
@@ -1008,8 +1008,10 @@ static Int TreeEntryC ( void )
 **  variables, and 'treeType' is assumed to be either 0 or 2,
 **
 **  Warning: 'factor' is not checked for being zero.
+**
+**  it returns 0 if everything worked, and 1 if an error arose.
```

Comment:
Maybe write ‘problem’ instead of ‘error’, like you did below. ('Error' to me kind of suggests that something _incorrect_ has happened, whereas this is more that something annoying has happened, if I understand correctly.).
```suggestion
**  it returns 0 if everything worked, and 1 if a problem arose.
```